istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -1373,7 +1458,27 @@ func ecdsConfigCmd(ctx cli.Context) *cobra.Command {
 	}
 
 	ecdsConfigCmd.PersistentFlags().StringVarP(&outputFormat, "output", "o", summaryOutput, "Output format: one of json|yaml|short")
+	ecdsConfigCmd.PersistentFlags().StringVarP(&labelSelector, "selector", "l", "", "Label selector")
 	ecdsConfigCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "", "Envoy config dump JSON file")
 
 	return ecdsConfigCmd
 }
+
+func getPodInfo(ctx cli.Context, kubeClient kube.CLIClient, labelSelector, pod string) (podName, podNamespace string, err error) {
+	var podNames []string
+	if labelSelector != "" {
+		podNames, podNamespace, err = getPodNameBySelector(ctx, kubeClient, labelSelector)
+		if err != nil {
+			return "", "", err
+		}
+		if len(podNames) > 1 {
+			fmt.Printf("Multiple pods found, use the first one: %s.%s\n\n", podNames[0], podNamespace)
```

Comment:
What's the use case for needing this new proxy-config command? And why would we just want the first pod found/returned in the case where multiple pod names exist for that label selector?